src/helpers.ts

```diff
@@ -315,3 +321,36 @@ export function parseNetworkConditions(args: string): NetworkConditions {
 
   return networkConditions;
 }
+
+// default stack trace limit
+const dstackTraceLimit = 10;
```

Comment:
Hmmm, interesting point. Do you think it provides value? In the past, I had used them when V8 async promise stack trace was not done correctly and even if you do increase the stack trace limit, its only going to show the Jest stack traces which seems not so useful to me. WDYT?

Reply:
I've previously used this on Deno and I use a similar option to debug errors on test diffs, although that's not exactly a stack trace, it's similar.

I don't think it's a very important change, it just came to mind because it was helpful for me in the past, although in very very few situations.

Reply:
Sounds good, if its useful let's do it. Let me follow up with a PR.